docs_src/design/adr/device-service/0011-DeviceService-Rest-API.md

```diff
@@ -96,13 +96,23 @@ A Reading represents a value obtained from a deviceResource. It contains the fol
 
 | Field name | Description
 | --- | ---
+| *device* | The name of the device
 | *name* | The name of the deviceResource
-| *valueType* | The type of the data
 | *origin* | A timestamp indicating when the reading was taken
 | *value* | The reading value
-| *mediaType* | (Only for Binary readings) The MIME type of the data
+| *valueType* | The type of the data
 | *floatEncoding* | (Only for floats and arrays of floats) The float representation in use
```

Comment:
remove

Reply:
Sorry, maybe I should've been more specific in my original comment. As we've removed the whole concept of float encoding, I think it would still be worthwhile to scrub any remaining references to it in your ADR.